src/views/homepage/components/chiclets/chiclets.module.css

```diff
@@ -4,30 +4,47 @@
  */
 
 .root {
-	margin-top: 72px;
+	margin-top: 122px;
+
+	@media only screen and (max-width: 1440px) {
+		margin-top: 72px;
+	}
+
+	@media (--dev-dot-mobile) {
+		margin-top: 32px;
+	}
 }
 
 .label {
-	/* composition */
-	composes: hds-typography-display-100 from global;
-
-	/* properties */
+	composes: hds-font-weight-bold from global;
 	color: var(--token-color-foreground-high-contrast);
-	font-weight: var(--token-typography-font-weight-medium);
-}
+	font-family: var(--token-typography-display-400-font-family);
+	font-size: var(--token-typography-display-400-font-size);
+	line-height: var(--token-typography-display-400-line-height);
+	margin: 0;
+	margin-bottom: 24px;
+	padding: 0;
 
-.nav {
-	margin-top: 30px;
+	@media (--dev-dot-mobile) {
+		font-family: var(--token-typography-display-300-font-family);
+		font-size: var(--token-typography-display-300-font-size);
+		line-height: var(--token-typography-display-300-line-height);
+		margin-bottom: 20px;
+	}
```

Comment:
🗒️ Unfortunately can't use `composes` to make these shifts in media queries, so I had to pull out the tokens and apply them directly.